remoter-server/src/dataIntegration.r

```diff
@@ -0,0 +1,109 @@
+################################################
+## STEP 6. Data integration
+#################################################
```

Comment:
Use the latest version of DataIntegration that can be found here (https://github.com/biomage-ltd/data-ingest/blob/Change-config/src/QC_helpers/dataIntegration.r), since the latest one support persisting the numPCs inside the seurat object